src/components/Tutorial/Assessments/Quiz.vue

```diff
@@ -114,10 +114,10 @@ export default {
         this.userChoices[correctChoice].checked
       ));
     },
-    ariaLiveText: ({ checkedIndex, choices }) => {
+    ariaLiveText: ({ checkedIndex, choices, $t }) => {
```

Comment:
Something here is crashing for me when I try to submit an answer for an assessment

Reply:
fixed it!